manheim_cloudmapper/ses/report.py

```diff
@@ -0,0 +1,179 @@
+import os
+import datetime
+import logging
+from premailer import transform
+from .ses import SES
+
+logger = logging.getLogger(__name__)
+
+
+class Report():
+
+    # Base path for CSS stylings, default for cloudmapper
+    BASE_PATH = '/opt/manheim_cloudmapper/web/css'
+
+    def __init__(self,
+                 report_source=(
+                     '/opt/manheim_cloudmapper/web/account-data/report.html'
+                 ),
+                 account_name=None, sender=None, recipient=None,
+                 region=None, ses_enabled=None):
+        """
+        Initialize the Cloudmapper report. Sets variables for email.
+
+        :param account_name: AWS account name
+        :type account_name: str
+        :param sender: Email address registered to SES that will send emails
+        :type sender: str
+        :param recipient: Email address to send emails to
+        :type recipient: str
+        :param region: AWS Region
+        :type region: str
+        :param ses_enabled: wheter the SES email functionality is enabled
+        :type ses_enabled: str
+        """
+        self.report_source = report_source
+        if account_name is None:
+            account_name = os.environ.get('ACCOUNT', None)
+
+        if sender is None:
+            sender = os.environ.get('SES_SENDER', None)
+
+        if recipient is None:
+            recipient = ('AWS SES <' +
+                         os.environ.get('SES_RECIPIENT', None) + '>')
+
+        if region is None:
+            region = os.environ.get('AWS_REGION', None)
+
+        if ses_enabled is None:
+            ses_enabled = os.environ.get('SES_ENABLED', None)
+
+        self.account_name = account_name
+        self.sender = sender
+        self.recipient = recipient
+        self.region = region
+        self.ses_enabled = ses_enabled
```

Comment:
It seems to me like this should be handled at a higher level, and we shouldn't even instantiate the class in the first place if SES isn't enabled...